packages/search/src/features/registration/birth/service.ts

```diff
@@ -142,7 +142,7 @@ async function createChildIndex(
     return
   }
 
-  await addEventLocation(body, BIRTH_ENCOUNTER_CODE, composition)
+  await addEventLocation(bundle, body, BIRTH_ENCOUNTER_CODE)
```

Comment:
@Nil20 Should this call be made from `createIndexBody` as it isn't directly related to the child?